<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CGameDlg</class>
 <widget class="QWidget" name="CGameDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_caidan">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>310</y>
     <width>75</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:none;
padding:11px;
border-radius:3px;
background:transparent;</string>
   </property>
   <property name="text">
    <string>菜单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_zanting">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>370</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:none;
padding:11px;
border-radius:3px;
background:transparent;</string>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_tishi">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>370</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:none;
padding:11px;
border-radius:3px;
background:transparent;</string>
   </property>
   <property name="text">
    <string>提示</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>471</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>40</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>122</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:none;
padding:11px;
border-radius:3px;
background:transparent;</string>
   </property>
   <property name="text">
    <string>音量-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>160</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:none;
padding:11px;
border-radius:3px;
background:transparent;</string>
   </property>
   <property name="text">
    <string>音量+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>200</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:none;
padding:11px;
border-radius:3px;
background:transparent;</string>
   </property>
   <property name="text">
    <string>静音/恢复</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>420</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:none;
padding:11px;
border-radius:3px;
background:transparent;</string>
   </property>
   <property name="text">
    <string>洗牌</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_daojv1">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>110</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:none;
padding:11px;
border-radius:3px;
background:transparent;</string>
   </property>
   <property name="text">
    <string>十字</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_daojv1_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>180</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:none;
padding:11px;
border-radius:3px;
background:transparent;</string>
   </property>
   <property name="text">
    <string>炸弹</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_daojv1_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>250</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:none;
padding:11px;
border-radius:3px;
background:transparent;</string>
   </property>
   <property name="text">
    <string>消同</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>121</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>剩余：1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>191</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>剩余：1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>251</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>剩余：1</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
